### PR TITLE
new feature: highlight current tag

### PR DESCRIPTION
**Pull request by yongbing-chen**
With "highlight current tag", user can quickly locating to the related tag in the TagInput component.
1. select a region in canvas in EditPage
2. double click on the region

the TagInput will scroll to the tag realated to the field you clicked.

highlight current tag

double click tag go to label page 
1. double click on a tag whitch realted label not in current page.
the page will switch to the correct page

double click tag show label page

### CODE REVIEW

File: src/react/components/common/imageMap/imageMap.tsx [hunk 1]
```diff
@@ -1078,8 +1094,8 @@ export class ImageMap extends React.Component<IImageMapProps> {
         this.initializeDrawnRegionLabelLayer();
         this.initializeDrawnRegionLayer();
         return [this.imageLayer, this.textVectorLayer, this.tableBorderVectorLayer, this.tableIconBorderVectorLayer,
-            this.tableIconVectorLayer, this.checkboxVectorLayer, this.drawRegionVectorLayer, this.labelVectorLayer,
-            this.drawnLabelVectorLayer];
+        this.tableIconVectorLayer, this.checkboxVectorLayer, this.drawRegionVectorLayer, this.labelVectorLayer,
```
Comment (stew-ro) [inline src/react/components/common/imageMap/imageMap.tsx:1097]: Is there a auto format tool that you're using? Sometimes there's so many style changes, it's hard to review the actual code for the feature.

Also, some of these style changes seem incorrect., Like here I think there should be an indent.

File: src/react/components/common/tagInput/tagInput.tsx [hunk 2]
```diff
@@ -510,14 +509,24 @@ export class TagInput extends React.Component<ITagInputProps, ITagInputState> {
...
     }
-
+    focusTag(tag: string) {
+        const tagItemRef = this.tagItemRefs.get(tag)?.getTagNameRef();
+        if (tagItemRef) {
+            tagItemRef.current.scrollIntoView({ behavior: "smooth" });
+            tagItemRef.current.classList.add("tag-item-highlight");
+            setTimeout(() => {
+                tagItemRef.current.classList.remove("tag-item-highlight");
+                console.log('remove("tag-item-highlight")');
```
Comment (stew-ro) [inline src/react/components/common/tagInput/tagInput.tsx:519]: remove console log

**Review by stew-ro — DISMISSED** [review]

File: src/react/components/common/tagInput/tagInput.tsx (hunk 2 above)
Comment (yongbing-chen) [reply src/react/components/common/tagInput/tagInput.tsx]: done

File: src/react/components/common/imageMap/imageMap.tsx (hunk 1 above)
Comment (yongbing-chen) [reply src/react/components/common/imageMap/imageMap.tsx:1097]: just format the code with the vs code format (Shift+Alt+F)

**Review by alex-krasn — APPROVED** [review]

**Review by stew-ro — APPROVED** [review]

**Pull request merged**